base/cloudbuild.yaml.in

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: ['build', '--tag=${IMAGE}', './base']
```

Comment:
possibly, but I think that's more an issue with `docker build --tag` (since the value provided is technically the entire image name plus the tag, not just the tag itself). the `IMAGE` value is used again after this when it's provided to the `--image` flag of the structure tests, which makes more sense.